<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editTransactionDialog</class>
 <widget class="QDialog" name="editTransactionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>221</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Transaction</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="backgroundColor" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777</width>
       <height>16777</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#backgroundColor{
background-color: white; 
border-radius: 8px;
border: 0.5px solid rgb(222, 221, 218);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <property name="spacing">
       <number>15</number>
      </property>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="transDescBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #E7E8F0;
padding: 0 8px;
font: 300 13px &quot;Montserrat&quot;;
border-radius: 2px;
border: 1px solid rgb(154, 153, 150);</string>
        </property>
        <property name="placeholderText">
         <string>Enter a brief description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" alignment="Qt::AlignVCenter">
       <widget class="QComboBox" name="transCatBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
	background-color: #E7E8F0;
	font: 300 13px &quot;Montserrat&quot;;
	color: black;
}</string>
        </property>
        <property name="placeholderText">
         <string>Select a category</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="closeBtn">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color: white;
border-radius: 5px;
border:1px solid rgb(222, 221, 218);
color: black;
font: 400 13px &quot;Montserrat&quot;;
}


QPushButton:pressed {
background-color: rgb(222, 221, 218);
}</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignVCenter">
       <widget class="QFrame" name="transOpt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #E7E8F0;
padding: 0 8px;
font: 300 13px &quot;Montserrat&quot;;
border-radius: 2px;
border: 1px solid rgb(154, 153, 150);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="optIncome">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator::unchecked{
border-radius: 5px;
background-color: white;
border: 1px solid black;
}
QRadioButton::indicator::checked{
border-radius: 5px;
background-color: black;
border: 1px solid white;
}

QRadioButton{border:0px solid black;}

QRadioButton::indicator{
width: 9px;
height: 9px;
border-style: inset;
border: 1px solid gray;
} </string>
           </property>
           <property name="text">
            <string>Income</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignVCenter">
          <widget class="QRadioButton" name="optExpenses">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator::unchecked{
border-radius: 5px;
background-color: white;
border: 1px solid black;
}
QRadioButton::indicator::checked{
border-radius: 5px;
background-color: black;
border: 1px solid white;
}

QRadioButton::indicator{
width: 9px;
height: 9px;
border-style: inset;
border: 1px solid gray;
} 

QRadioButton{border:0px solid black;}</string>
           </property>
           <property name="text">
            <string>Expenses</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="transDateBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #E7E8F0;
color: black;
font: 300 13px &quot;Montserrat&quot;;</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="displayFormat">
         <string>dd MMMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="transAmntBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #E7E8F0;
color: black;
font: 300 13px &quot;Montserrat&quot;;
padding: 0 14px;</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="prefix">
         <string>$ </string>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="transUpdateBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:  #2a59ff;
color: white;
border-radius: 3px;
font: 500 13px &quot;Montserrat&quot;;}

QPushButton:hover {
    border-style: inset;
	border: 1px solid gray;
}

QPushButton:pressed {
	
	background-color: #0035d0;
    border-style: inset;
	border: 1px solid gray;
}</string>
        </property>
        <property name="text">
         <string>Update Transaction</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>25</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="deleteButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{background-color: white;
border-radius: 5px;
border:1px solid rgb(222, 221, 218);
color: black;
font: 400 13px &quot;Montserrat&quot;;
}


QPushButton:pressed {
background-color: rgb(255, 179, 179);
}</string>
          </property>
          <property name="text">
           <string> Delete</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/resources/remove.png</normaloff>:/resources/remove.png</iconset>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="transReportLabels">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="transInfoLabelWidget">
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="transInfoLabel">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 300 12px &quot;Montserrat&quot;;
color: black;
background-color: rgb(246, 245, 244);
border-radius: 2px;</string>
              </property>
              <property name="text">
               <string>Edit the transaction using the above form.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="transSuccessLabelWidget">
           <layout class="QGridLayout" name="gridLayout_6">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="transSuccessLabel">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>167777</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 400 12px &quot;Montserrat&quot;;
color: green;
background-color: rgb(220, 255, 218);
border-radius: 2px;</string>
              </property>
              <property name="text">
               <string>Sucess Entry</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="transErrorLabelWidget">
           <layout class="QGridLayout" name="gridLayout_7">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="transErrorLabel">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 400 12px &quot;Montserrat&quot;;
color: red;
background-color: rgb(255, 196, 196);
border-radius: 2px;</string>
              </property>
              <property name="text">
               <string>Invalid Entry</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="dateLabel_6">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 500 14px &quot;Montserrat&quot;;
background-color:white;</string>
        </property>
        <property name="text">
         <string>Edit Transaction</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
